config/main.py

```diff
@@ -795,6 +797,7 @@ def _wait_until_clear(tables, interval=0.5, timeout=30, verbose=False):
         empty = (non_empty_table_count == 0)
     if not empty:
         click.echo("Operation not completed successfully, please save and reload configuration.")
+        WAIT_UNTIL_CLEAR_STATUS = 1
     return empty
```

Comment:
is it possible to make use of this return value instead of a global variable?

Reply:
Agree, good idea, will try it.